spec/features/bars/user_adds_bar_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+feature "authenticated user adds a bar" do
+  let(:new_user) { User.create(email: "mckelvey.matt@gmail.com", password: "12345678") }
+
+  scenario "successfully creates a new bar" do
+
+    new_user
+    visit user_session_path
+    fill_in 'Email', with: 'mckelvey.matt@gmail.com'
+    fill_in 'Password', with: '12345678'
+    click_button "Log in"
+    save_and_open_page
+    click_button "Add New Bar"
+    fill_in 'Name', with: 'Beantown Pub'
+    fill_in 'Description', with: 'After hours cocktail bar with pool.'
+    click_button "Add Bar"
+
+    expect(page).to have_content('Beantown Pub')
+  end
+
+  scenario "submits form without a name or description" do
+
+    new_user
+    visit user_session_path
+    fill_in 'Email', with: 'mckelvey.matt@gmail.com'
+    fill_in 'Password', with: '12345678'
+    click_button "Log in"
```

Comment:
You're using the same code to log in the user for both `scenario`s here! I'd recommend pulling that out into a `before` block like so:

```
feature "..." do
  before(:each) do 
    # code to sign in a user
  end
  
  scenario "..." do
    ...
  end

  scenario "..." do
    ...
  end
end
```